scenario/GovernanceScenario.ts

```diff
@@ -157,9 +157,9 @@ scenario('add new asset',
   });
 
 scenario(
-  'execute Mumbai governance proposal',
+  'execute Polygon governance proposal',
   {
-    filter: async ctx => ctx.world.base.network === 'mumbai'
+    filter: async ctx => ctx.world.base.network === 'mumbai' || ctx.world.base.network === 'polygon'
```

Comment:
we could use the `matchesDeployment` helper fn here

Reply:
Good rec. I changed it to use `isBridgedDeployment` instead since I think that's easier to maintain.

Reply:
`isBridgedDeployment` will be true for other L2 deployments (like Optimism in the near future), right? but these tests only apply to Mumbai and Polygon

Reply:
ah you're absolutely right

Reply:
Done